### DISCUSSION TITLE
irssi like UI?

### DISCUSSION THREAD

**Discussion by hemna** [Textualize/textual#4486] [General]
Hey guys,
  I'm looking at writing a python app for the command line that has a need for an irssi like interface for a chat with multiple destinations via tabs and a footer with the status of the current chat and the last line the input area for user input.

https://en.wikipedia.org/wiki/Irssi#/media/File:Irssi_1.2.3_screenshot.png

Is there a way to do that with textual?

**Comment by davep** [comment]
I don't know the app you mention, but I can't see why you couldn't develop such a UI using Textual.
